### ISSUE TITLE
Build a simple UI for handling rules and expressions

### ISSUE THREAD

**Issue by luke-bhan** [ayazhafiz/slide#115]
I think we should start thinking about an interface for our platform. Some small website where you can add rules (Set a default set of rules) with just checking boxes and then type your expression similar to wolfram and we simplify. This certainly can be on the backlog, but thoughts @ayazhafiz 
@timothycobrien

**Comment by ayazhafiz** [comment]
This would be nice, I tried this for sherpa_41 but it didn't go well. That was also a giant C++ app though, so I feel like this would be a bit easier. Sympy has a similar thing: https://www.sympygamma.com/input/?i=DiracDelta%28x**2+%2B+x+-+2%29.simplify%28x%29. Maybe google app engine, heroku, or glitch are the easiest here.

I agree this should be on the backlog though. We still need a lot of work to get the simplifier to work very well, but this is a great thing to have.

**Comment by ayazhafiz** [comment]
@luke-bhan do you want to take this one?

**Comment by luke-bhan** [comment]
yes. I'd like to do this this weekend if nobody is working on it. May take a while, very unfamiliar with these frameworks, so good learning expereince

**Comment by ayazhafiz** [comment]
Heck yes, do it!

________________________________
От: Luke Bhan <notifications@github.com>
Отправлено: Friday, June 26, 2020 8:25:46 PM
Кому: ayazhafiz/slide <slide@noreply.github.com>
Копия: hafiz <ayaz.hafiz.1@gmail.com>; Mention <mention@noreply.github.com>
Тема: Re: [ayazhafiz/slide] Build a simple UI for handling rules and expressions (#115)


yes. I'd like to do this this weekend if nobody is working on it. May take a while, very unfamiliar with these frameworks, so good learning expereince @ayazhafiz<https://github.com/ayazhafiz>

**Comment by luke-bhan** [comment]
https://slideuiapp.ue.r.appspot.com/

**Comment by luke-bhan** [comment]
It should just say hello world for now. But app engine is setup :)

**Comment by ayazhafiz** [comment]
yooo! hell yeah!
